ow_power_system/src/power_system_node.cpp

```diff
@@ -90,10 +39,19 @@ int main(int argc, char* argv[]) {
 
```

Comment:
onewatt.csv is incorrect.  Use the named constant for the default file I suggested above.

Reply:
Not entirely sure what this comment is in reference to, but I believe I removed all references to onewatt.csv. I will make the change to a 'bool' variable and named constant in a new commit

Reply:
onewatt.csv still shows up on line 37 of my version of this file, and I have pulled your latest branch.   Maybe your local copy is not committed/pushed?

Reply:
Ahh nope, you're right! I missed that one bc it's a longer line, so the portion containing that reference had been cut off in my editing window. Good catch!

Reply:
Resolved